command/operator_debug.go

```diff
@@ -512,7 +512,7 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 	}
 
 	c.opts = &api.QueryOptions{
-		Region:     c.Meta.region,
+		Region:     c.Meta.Region(),
```

Comment:
This PR fixes the reported bug, but why are we passing the region like this at all in an API request? I think `operator debug` is the only command to do so. This should already go to the right region by virtue of how the API client gets constructed.

```suggestion
```

Reply:
good point. I believe here in `QueryOptions` an empty region is ignored, and specifying it is redundant, because yes it's a part of the API client, so may as well drop this line.

for clarity, the actual bugfix is passing the proper region into `filterServerMembers` below.

and now I'm realizing that `NOMAD_NAMESPACE` is unaffected by this bug, and its only shortcoming was not being `c.Ui.Output` properly.